src/pycram/orm/utils.py

```diff
@@ -0,0 +1,112 @@
+import traceback
+
+import sqlalchemy
+import pycram.orm.base
+from pycram.designators.action_designator import *
+from pycram.designators.object_designator import *
+
+
+def logDB(in_session, filename, b_write_to_console=False):
```

Comment:
no abbreviations in variables, functions, etc..
Also no camel case for function names and variables.

Reply:
Also type hint the parameters and return values